cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -103,11 +103,6 @@ public String getFormat() {
     public Network importData(ReadOnlyDataSource ds, NetworkFactory networkFactory, Properties p) {
         CgmesModel cgmes = CgmesModelFactory.create(ds, boundary(p), tripleStore(p));
         Network network = new Conversion(cgmes, config(p), activatedPostProcessors(p), networkFactory).convert();
```

Comment:
As we have moved the management of the extensions inside the conversion object, we could return directly the created the network without storing it in an intermediate variable